app/navigation/router.js

```diff
@@ -92,13 +105,49 @@ class Router extends React.Component {
     };
 
     render = () => {
+        const {
+            leftDrawerOpen,
+            leftDrawerRoute,
+            rightDrawerOpen,
+            rightDrawerRoute
+        } = this.props.navigation;
+
+        let leftDrawerContent;
+        if (leftDrawerRoute) {
+            leftDrawerContent = this.renderRoute(leftDrawerRoute);
+        }
+
+        let rightDrawerContent;
+        if (rightDrawerRoute) {
+            rightDrawerContent = this.renderRoute(rightDrawerRoute);
+        }
+
         return (
-            <NavigationExperimental.Transitioner
-                style={{flex: 1}}
-                navigationState={this.props.navigation}
-                render={this.renderTransition}
-                configureTransition={this.configureTransition}
-            />
+            <Drawer
```

Comment:
Very smart to put this here, this seems to me a great decision.

One question: when the stack is showing the drawers there's still going to be a scene in the bottom not completely overlapped right?